CaughtUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="a0y2v6pCXfyvHgf6HUU8KPdaJ" />
     </application>
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Agreed, we should leave it. It's required even for API 23, it just might not be enough. Necessary but not sufficient, etc.